extension/scripts/features/last-action/ttCompanyLastAction.js

```diff
@@ -111,6 +114,8 @@
 			});
 		}
 		list.classList.add("tt-modified");
+
+		triggerCustomListener(EVENT_CHANNELS.FEATURE_ENABLED, { name: feature.name });

```

Comment:
The feature shouldn't send out this event, the feature manager will handle it if you register it properly.

After the `requirements`, you can add an `options` object to registering the feature. Settings `triggerCallback` to true will trigger this listener.